attest/attest.go

```diff
@@ -40,9 +40,9 @@ type AttestationDoc struct {
 }
 
 type AttestationUserData struct {
-	FuncHash      []byte `json:"func_hash"`
-	KeyPolicyHash []byte `json:"key_policy_hash"`
-	CapeKey       []byte `json:"key"`
+	FuncChecksum []byte `json:"func_checksum"`
+	KeyChecksum  []byte `json:"key_checksum"`
```

Comment:
Is this still a checksum of the policy, or is it a checksum of the key?

Reply:
checksum of the key policy